drf_polymorphic/extensions.py

```diff
@@ -0,0 +1,85 @@
+from drf_spectacular.extensions import OpenApiSerializerExtension
+from drf_spectacular.plumbing import ResolvedComponent, build_object_type
+
+from .serializers import PolymorphicSerializer
+
+
+class PolymorphicSerializerExtension(OpenApiSerializerExtension):
+    """
+    drf-spectacular extension for `PolymorphicSerializer`
```

Comment:
```suggestion
    drf-spectacular extension for :class:`PolymorphicSerializer`
```
RST is picky about this :-) use the `class` directive or double backticks

Reply:
Ok! Thanks!